src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -235,6 +235,13 @@ public string HubName
         /// </summary>
         public AppLeaseOptions AppLeaseOptions { get; set; } = AppLeaseOptions.DefaultOptions;
 
+        /// <summary>
+        /// Option to control the receive message size  in bytes of the grpc client, which is used by Durable Funsiont c# Isolated and Java.
+        /// Defaults to 4,194,304 (4 MB).
+        /// In .NET 6.0 or later, setting this value to <c>null</c> removes the receive message size limit.
+        /// </summary>
+        public int? MaxGrpcMessageSize { get; set; } = 4194304;
```

Comment:
Consider naming this `MaxGrpcMessageSizeInBytes` to make it clear what the size unit is.

Reply:
Seems like we already make maxReceiveMessageSize max [there](https://github.com/Azure/azure-functions-durable-extension/blob/e211fbbe3a28cb61932f386dee2fbbd98cc05439/src/WebJobs.Extensions.DurableTask/LocalGrpcListener.cs#L64C70-L64C71)